socorro/lib/util.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, msg, ret=None):
 
 def wait_time_generator():
     """Return generator for wait times."""
-    yield from [2, 2, 2, 2, 2]
+    yield from [1] * 5
```

Comment:
Changed this from 2, 2, 2, 2, 2 to 1, 1, 1, 1, 1.

There was another case in sentry where the crashstorage code was retrying pushing something to AWS S3. We don't need to wait 2 seconds between iterations.